views/js/runner/api/itemRunner.js

```diff
@@ -418,7 +418,20 @@ define([
                     responses = provider.getResponses.call(this);
                 }
                 return responses;
-           }
+           },
+
+            /**
+             * Render modalFeedbacks according to itemSession into the current item object
+             *
+             * @param feedbacks
```

Comment:
The most important is missing : what `feedback` is supposed to contain ? And `itemSession` ? Does the `done` callback have parameters ?